REST_API_DOCS.md

```
@@ -0,0 +1,1205 @@
+```json
+{
+  "status": "error",
+  "message": "Authentication required"
+}
+```
+
+---
+
+### { 내가 쓴 게시글 목록 }
+
+**GET** `/users/me/posts` 
```

markdownlint-cli2 (0.18.1)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)